app/dashboard/views.py

```diff
@@ -758,25 +758,38 @@ def users_fetch(request):
     skills = request.GET.get('skills', '')
     limit = int(request.GET.get('limit', 10))
     page = int(request.GET.get('page', 1))
-    order_by = request.GET.get('order_by', '-actions_count')
+    order_by = request.GET.get('order_by', 'leaderboard_ranks__rank')
     bounties_completed = request.GET.get('bounties_completed', '').strip().split(',')
     leaderboard_rank = request.GET.get('leaderboard_rank', '').strip().split(',')
     rating = int(request.GET.get('rating', '0'))
     organisation = request.GET.get('organisation', '')
 
     user_id = request.GET.get('user', None)
     if user_id:
-        profile = Profile.objects.get(id=int(user_id))
+        profile = User.objects.get(id=int(user_id))
```

Comment:
Lets not call an instance of a `User` object `profile` so we can avoid confusion here

Reply:
Yeah. makes sense. It's only that in the loop below we were using the variable `user` to loop through the user list. I will rename this as `current_user`